### DISCUSSION TITLE
Unable to perform direct edit operation on InDesign files

### DISCUSSION THREAD

**Discussion by PGS-gh** [pimcore/pimcore#9958] [Q&A / Support]
**Steps**:
1.  Open any InDesign file(indd extensions) with Direct Edit plugin. It successfully opens in Adobe InDesign.
2.  Now, edit and save changes.

**Actual Behavior**:
After editing and saving changes, it opens a pop-up and try saving the updated InDesign file in the local system only and not in Pimcore.

**Expected Behavior**
After editing and saving changes, it should close the Adobe InDesign and ask to apply changes in Pimcore. But Pimcore Direct Edit Client application doesn't able to detect the Adobe InDesign save change. 

**_What could be the issue? 
Is it something to do with the Adobe InDesign internal functionality(as direct edit is working fine for word, ppt, adobe illustrator etc..)?_**

This is what I get after saving the indesign files
![Adobe Indesign Issue](https://user-images.githubusercontent.com/85244913/128451200-70e3e112-2f9b-4c38-87ea-70137778a961.png)

@fashxp @brusch can you guys please provide any inputs for this?

**Comment by PGS-gh** [comment]
Not a _Direct Edit_ plugin issue but of Adobe _InDesign_.
If you try to open an _InDesign_ file of older version in a newer version(installed in your local system) through _Direct Edit_ plugin, it first tries to convert it into newer version and asks to save in the local system first.

**Reply by fashxp** [reply]
so there is nothing we can do about?

**Reply by fashxp** [reply]
for the future ... please add issues for enterprise extensions to the corresponding repositories, you should already have access to them.
